### PR TITLE
[Improve][Docs] Add tuning guide for doris sink in streaming mode

### PR DESCRIPTION
**Pull request by dailai**
### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/Doris.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ The internal implementation of Doris sink connector is cached and imported by st
 | custom_sql                     | String  | no       | -                            | When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL parameter. This parameter usually fills in a SQL that can be executed. SQL will be executed before synchronization tasks.                                                                             |
 | doris.config                   | map     | yes      | -                            | This option is used to support operations such as `insert`, `delete`, and `update` when automatically generate sql,and supported formats.                                                                                                                                              |
 
+### Note
+Appropriately increasing the value of `sink.buffer-size` and `doris.batch.size` can increase the write performance. <br>
+In stream mode, if the `doris.batch.size` and `checkpoint interval` are both configured with a large value, The last data to arrive may have a large delay(The delay time is the checkpoint interval). <br>
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Doris.md]:
```suggestion
In stream mode, if the `doris.batch.size` and `checkpoint.interval` are both configured with a large value, The last data to arrive may have a large delay(The delay time is the checkpoint interval). <br>
```

Comment (dailai) [reply docs/en/connector-v2/sink/Doris.md]: Done

File: docs/en/connector-v2/sink/Doris.md [hunk 2]
```diff
@@ -51,6 +51,12 @@ The internal implementation of Doris sink connector is cached and imported by st
 | custom_sql                     | String  | no       | -                            | When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL parameter. This parameter usually fills in a SQL that can be executed. SQL will be executed before synchronization tasks.                                                                             |
 | doris.config                   | map     | yes      | -                            | This option is used to support operations such as `insert`, `delete`, and `update` when automatically generate sql,and supported formats.                                                                                                                                              |
 
+### Note
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Doris.md]: It's weird to put `Note` in Sink Options part. 
How about add new part named `Tuning Guide` before `Task Example` part?
We can extend the `Tuning Guide` to all connectors next.

Comment (dailai) [reply docs/en/connector-v2/sink/Doris.md]: Ok

**Review by Hisoka-X — DISMISSED** [review]
LGTM. Thanks @dailai

File: docs/zh/connector-v2/sink/Doris.md [hunk 3]
```diff
@@ -147,6 +147,13 @@ CREATE TABLE IF NOT EXISTS `${database}`.`${table_name}`
 
 支持的格式包括 CSV 和 JSON。
 
+## 调优指南
+
+适当增加`sink.buffer-size`和`doris.batch.size`的值可以提高写性能。< br >
+在流模式下，如果`doris.batch.size`和`checkpoint.interval`都配置为较大的值，最后到达的数据可能会有较大的延迟(延迟的时间就是检查点间隔的时间)。< br >
```
Comment (wuchunfu) [inline docs/zh/connector-v2/sink/Doris.md]: @dailai Please modify `< br >` to `<br>`

Comment (dailai) [reply docs/zh/connector-v2/sink/Doris.md]: Done

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**